src/Form/SubscriptionForm.php

```diff
@@ -90,6 +90,7 @@ public function buildForm(array $form, FormStateInterface $form_state, stdClass
           '#title' => $contact_field['label'],
           '#description' => $contact_field['description'],
           '#required' => !empty($contact_field['required']),
+          '#default_value' => $this->getRequest()->get($contact_field_name) ?? NULL,
```

Comment:
The `?? NULL` makes no sense here. It will replace `NULL` with `NULL`.

Reply:
Never trust anyone :wink: de6b5591718e89e80206d8b4dae16fefaaad9b14